jupyterlab_spellchecker/dictionaries.py

```diff
@@ -0,0 +1,53 @@
+import os
+import json
+
+from pathlib import Path
+
+from jupyter_core.paths import jupyter_path
+from notebook.utils import url_path_join
+
+def _resolve_language(dp, lang):
+    code = lang['code']
+    lang['path'] = dp
+
+    return lang
+
+
+def _scan_for_dictionaries(data_path):
+    p = Path(data_path)
+    lang_jsons = list(p.glob('**/lang.json'))
```

Comment:
Two thoughts:
- we should decide on a location within jupyter paths (e.g. lab/dictionaries) so that it does no search all jupter files which might take some time
- maybe in addition to jupyter_paths we should use also search standard locations, e.g.:

> - /usr/share/hunspell (Linux only)
> - /usr/share/myspell (Linux only)
> - /usr/share/myspell/dicts (Linux only)
> - / (Mac only)
> - /System/Library/Spelling (Mac only)
> - C:\ (Windows only)

(copied from https://atom.io/packages/spell-check)

This would imply that we would need to manage without having `lang.json`. A simple solution would be to maintain a list of language code -> language name mappings; we could also use [babel](https://github.com/python-babel/babel) if we do not want to maintain the mapping ourselves, but maybe having no dependencies would be best..

Reply:
Also the paths for Mac and Windows are too generic. We cannot just search the whole drive.a

Reply:
I will give it a shot.

Reply:
See #74.